lib/hostip.c

```diff
@@ -244,6 +244,7 @@ void Curl_hostcache_prune(struct Curl_easy *data)
   if(data->share)
     Curl_share_lock(data, CURL_LOCK_DATA_DNS, CURL_LOCK_ACCESS_SINGLE);
 
+  now = time(NULL);
```

Comment:
Is there really a point in saving the old call as well? Making two calls for the same thing seems redundant and `time(NULL);` is used stand-alone in several places in the code already!

Reply:
Thanks for pinpointing this, `time()` is returning the current time with `time(NULL)` already, no point in doing it twice indeed!